tox.ini

```diff
@@ -0,0 +1,22 @@
+[tox]
+envlist = flake8, {py27,py35,py36}-django{10,11}, {py35,py36}-django20
+skip_missing_interpreters = True
+
+[testenv]
+changedir={toxinidir}
+commands=
+    ./test.sh
```

Comment:
Can you document why you're not simply including the contents of `test.sh` here? Is it because of `set -e` (and how does this change the default behaviour)?

Reply:
`set -e` stops execution on error. I'm using `test.sh` to be able to use pipes: under `commands=` you can't use them.

Reply:
Thanks!